CMakeLists.txt

```diff
@@ -15,7 +15,10 @@ include(GNUInstallDirs)
 set(HYPRLAND_VERSION ${VER})
 set(PREFIX ${CMAKE_INSTALL_PREFIX})
 set(INCLUDEDIR ${CMAKE_INSTALL_INCLUDEDIR})
+set(bindir ${CMAKE_INSTALL_BINDIR})
+set(prefix ${CMAKE_INSTALL_PREFIX})
```

Comment:
Oh, we already seem to have PREFIX. Let's not duplicate things then, just make bindir -> BINDIR and remove prefix.

Reply:
A moment